src/routes/Gigs/index.js

```diff
@@ -1,14 +1,58 @@
 import React from 'react'
+import List from '../../components/List';
+import QueryContainer from '../../containers/QueryContainer';
+import { GET_GIGS } from "../../queries/index"
+import { Link } from 'react-router-dom';
+import EmptyMessage from '../../components/EmptyMessage/EmptyMessage';
 
-class Gigs extends React.Component {
-  render() {
-    return (
-      <div className='container'>
-        <h2>Gigs 🗓️</h2>
-        <p>Coming soon...</p>
-      </div>
-    )
-  }
+const Gigs = () => {
+
+  return (
+    <div className='container'>
+
+      <QueryContainer
+        query={GET_GIGS}
+        entityName='gigs'
+        render={(queryData) => {
+
+          const hasGigs = (queryData && queryData !== null) && queryData.length;
+          const sortedQueryData = queryData.sort((a, b) => new Date(a.releaseDate) - new Date(b.releaseDate)).reverse();
+
+          return (
+            <div>
+
+              <div className='heading-with-btns'>
+
+                <div>
+                  <h2>Gigs 🗓️</h2>
+                </div>
+
+                <div className='action-btns'>
+                  <Link to='gigs/create' className='btn'>Create</Link>
+                </div>
+              </div>
```

Comment:
after getting the latest with master, we can use `<PageHeader />` here

```jsx
<PageHeader
  heading='Gigs 🗓️'
  entityCollection='gigs'
  renderCreateButton
/>
```

Reply:
ok!